libs/common/grpc.go

```diff
@@ -162,15 +171,15 @@ func handleOrganizationIDForAuthFunc(ctx context.Context) (context.Context, erro
 	}
 
 	if !hwutil.Contains(claims.Organizations, organizationID) {
-		zlog.Ctx(ctx).Info().Str("organizationID", organizationID.String()).Msg("organization in header was not part of claims")
+		log.Info().Str("organizationID", organizationID.String()).Msg("organization in header was not part of claims")
 		return nil, status.Errorf(codes.Unauthenticated, "no access to this organization")
 	}
 
 	ctx = context.WithValue(ctx, organizationIDKey{}, organizationID)
 
 	// Append organizationID to the logger
-	log := zlog.Ctx(ctx).With().Str("organizationID", organizationID.String()).Logger()
-	return log.WithContext(ctx), nil
+	loggerWithOrganizationID := zlog.Ctx(ctx).With().Str("organizationID", organizationID.String()).Logger()
```

Comment:
might need some attention

Reply:
Can you elaborate on that? The code is not _very nice_, but it's the same as above for the `userID`.

Reply:
could have been just
```go
log = log.With().Str("organizationID", organizationID.String()).Logger()
return log.WithContext(ctx), nil
```